modules/subscriptions.php

```diff
@@ -741,11 +756,48 @@ function set_cookies( $subscribe_to_post = false, $post_id = null, $subscribe_to
 		}
 	}
 
+	public static function register_block_type() {
+		if ( ! function_exists( 'register_block_type' ) ) {
+			return;
+		}
+		register_block_type( 'jetpack/subscription-form', array(
+			'render_callback' => 'jetpack_do_subscription_form'
+		) );
+	}
+
+	public static function enqueue_block_editor_assets() {
+		wp_register_script(
+			'jetpack-block-subscription-form',
+			plugins_url( 'subscriptions/block.js', __FILE__ ),
+			array( 'wp-blocks', 'wp-element' )
+		);
+		wp_enqueue_script( 'jetpack-block-subscription-form' );
+		wp_localize_script( 'jetpack-block-subscription-form', 'jpSubBlockI18n', array(
+			'Subscription Form' => __( 'Subscription Form', 'jetpack' ),
+			'Subscribe to Blog via Email' => __( 'Subscribe to Blog via Email', 'jetpack' ),
+			'Enter your email address to subscribe to this blog and receive notifications of new posts by email.'
+				=> __( 'Enter your email address to subscribe to this blog and receive notifications of new posts by email.', 'jetpack' ),
+			'Email Address' => __( 'Email Address', 'jetpack' ),
+			'Subscribe' => __( 'Subscribe', 'jetpack' ),
+			"Success! An email was just sent to confirm your subscription. Please find the email now and click 'Confirm Follow' to start subscribing."
+				=> __( "Success! An email was just sent to confirm your subscription. Please find the email now and click 'Confirm Follow' to start subscribing.", 'jetpack' ),
+			'Join %s other subscribers' => __( 'Join %s other subscribers', 'jetpack' ),
```

Comment:
Sorry, but here we really need to use `_n`. I'd say we just return a string from the subscriber count endpoint, rather than a number.